libyear-gradle-plugin/src/functionalTest/kotlin/com/libyear/LibYearPluginTest.kt

```diff
@@ -36,9 +36,35 @@ internal class LibYearPluginTest {
       .extracting { it?.outcome }.isEqualTo(TaskOutcome.FAILED)
   }
 
-  private fun withGradleRunner() = GradleRunner.create().apply {
+  @Test
+  fun testMaxTransitiveDepth0() {
```

Comment:
Thank you for adding the test cases. I think they exercise the code under test very well.